src/PlaceAutocompleteField.vue

```diff
@@ -11,9 +11,8 @@
             autocomplete="no"
             @blur="onBlur"
             @focus="onFocus"
-            @input="$emit('input', query)">
-            <activity-indicator v-if="showActivityIndicator" size="xs" type="spinner"/>
-        </input-field>
+            @input="$emit('input', query)" />
+        <activity-indicator v-if="showActivityIndicator" size="xs" type="spinner"/>
```

Comment:
Where did the closing input tag go?

Reply:
It is on line 14 at the end, do you want it on a new line?

Reply:
Oops, never mind.  

I'm using https://github.com/ds300/patch-package to patch it myself for now since I think these PR's are gonna sit for sometime and maybe this project is somewhat abandoned.  Thanks!